Documentation/proposals/202403-prometheus-operator-cli.md

```diff
@@ -0,0 +1,90 @@
+# Prometheus Operator CLI
+
+- Owners:
+  - [Nicolas Takashi](https://github.com/nicolastakashi)
+- Related Tickets
+  - [#6423](https://github.com/prometheus-operator/prometheus-operator/issues/6423)
+- Other docs:
+  - N/A
+
+This document proposes a new CLI tool to manage Prometheus Operator resources, allowing users to create, update, and delete Prometheus Operator resources using a simple CLI interface as well as
+troubleshoot and debug Prometheus Operator resources.
+
+# Why
+
+During KubeCon Paris 2024, we received feedback from users that managing Prometheus Operator resources is difficult and error-prone. Users reported that they often encounter issues when creating, updating, and deleting Prometheus Operator resources, and that troubleshooting and debugging Prometheus Operator resources is challenging, below you can find some of the issues that users reported:
+
+## Why my Prometheus resource is not being created?
+
+People were struggling to troubleshoot why their Prometheus resources were not being created. They were not sure if the issue was with the Prometheus Operator or with the Prometheus resource itself.
+After long investigation, they found out that the Prometheus Operator were only watching the namespace where the Prometheus Operator was deployed, and the Prometheus resource was being created in a different namespace.
+
+## Why my ServiceMonitor is not being created?
+
+People were struggling to troubleshoot why their ServiceMonitor resources were not being created. They were not sure if the issue was with the Prometheus Operator or with the ServiceMonitor resource itself.
```

Comment:
> why their ServiceMonitor resources were not being created

Do you mean that it's hard to understand why a pod isn't scraped by Prometheus rather?